python_modules/dagster/dagster/_core/execution/plan/active.py

```diff
@@ -305,6 +317,19 @@ def get_steps_to_execute(
             if limit is not None and len(batch) >= limit:
                 break
 
+            if "pool" in step.tags and increment:
```

Comment:
Could just move this part out to a separate function for now, but put it here since all the checks for concurrency seem to be done here..